tests/phpunit/app/Features/AttachmentTest.php

```diff
@@ -44,17 +54,33 @@ public function testOptionDefaultEnabled(): void
 		$this->assertTrue(Option::isOn('attachment_page'));
 	}
 
-	/** @testdox should update the option */
+	/** @testdox should update the option and `wp_attachment_pages_enabled` */
 	public function testOptionUpdated(): void
 	{
-		Option::update('attachment_page', true);
+		$wpVersion = $GLOBALS['wp_version'];
 
-		$this->assertEquals('1', get_option('wp_attachment_pages_enabled'));
-		$this->assertTrue(Option::isOn('attachment_page'));
+		if (version_compare($wpVersion, '6.4', '>=')) {
```

Comment:
We could have a method: `function isCoreVersionAtLeast(string $version)`

Reply:
@szepeviktor :) I'm fine with this approach, at least for now. I'm not in a rush to create an abstraction too early.